corehq/apps/api/resources/v0_4.py

```diff
@@ -375,7 +374,8 @@ class Meta(CustomResourceMeta):
 class BaseApplicationResource(CouchResourceMixin, HqBaseResource, DomainSpecificResourceMixin):
 
     def obj_get_list(self, bundle, domain, **kwargs):
-        return sorted(get_apps_in_domain(domain, include_remote=False), key=attrgetter("date_created"))
+        return sorted(get_apps_in_domain(domain, include_remote=False),
+                      key=lambda app: getattr(app, 'date_created') or datetime.min)
```

Comment:
i wasn't actually sure whether `min` or `max` made more sense here, but I think `min` is more in line with the intended behavior (creation order)

Reply:
Did you intend to specify a default for this attribute if it's not set? `getattr` with two args will raise `AttributeError` if the attribute does not exist. This is the same behavior as `attrgetter`, so I'm assuming you were not handling the case where the attribute is not set.

```suggestion
                      key=lambda app: app.date_created or datetime.min)
```

Reply:
ah yeah that wasn't intentional. it's a couch property so should always be there. https://github.com/dimagi/commcare-hq/pull/29532